tests/integrationtest/t/planner/core/integration.test

```diff
@@ -2040,6 +2041,7 @@ CREATE TABLE t(col1 enum('p5', '9a33x') NOT NULL DEFAULT 'p5',col2 tinyblob DEFA
 drop table if exists t;
 create table t (a int, b int, c int, index ia(a));
 select  * from t t1 join t t2 on t1.b = t2.b join t t3 on t2.b=t3.b join t t4 on t3.b=t4.b where t3.a=1 and t2.a=2;
+# TODO: The result is correct but does not exactly same.
 select plan from information_schema.statements_summary where SCHEMA_NAME = 'test' and STMT_TYPE = 'Select';
```

Comment:
I think the schema_name should be `planner__core__integration` instead of test.

Reply:
I check the code and find this test can't be done in integration test.
The origical code is here https://github.com/pingcap/tidb/pull/41461/files#diff-fa42dbc5f88d5aef4ee1cb7452ee4497fc545324b76bee0e44f48e5f1af9f512R5213-R5251